.github/workflows/phpunit-sqlite.yml

```diff
@@ -0,0 +1,103 @@
+# SPDX-FileCopyrightText: Nextcloud contributors
+# SPDX-License-Identifier: AGPL-3.0-or-later
+name: PHPUnit
```

Comment:
todo: change name to `name: PHPUnit SQLite` as in the original workflow

Reply:
changed